readthedocs/projects/models.py

```diff
@@ -925,6 +929,21 @@ class Meta(object):
 
     objects = HTMLFileManager()
 
+    @cached_property
+    def json_file_path(self):
+        basename = os.path.splitext(self.name)[0]
+        full_path = self.project.get_production_media_path(type_='json',
+                                                           version_slug=self.version.slug,
+                                                           include_file=False)
+
+        file_path = find_file(basename=basename, pattern='*.fjson', path=full_path)
+        return file_path
+
+    @cached_property
```

Comment:
How long is this cached for? Will we really see value in caching it vs. the tradeoff of keeping a bunch of JSON in memory?